Vagrantfile

```diff
@@ -163,6 +167,23 @@ def suse_common(config)
     java_package: "java-1_8_0-openjdk-devel")
 end
 
+def sles_common(config)
+  provision(config,
+    update_command: "zypper --non-interactive list-updates",
+    update_tracking_file: "/var/cache/zypp/packages/last_update",
+    install_command: "zypper --non-interactive --quiet install --no-recommends",
+    java_package: "java-1_8_0-openjdk-devel",
+    extra: <<-SHELL
+      zypper rr systemsmanagement_puppet || true
```

Comment:
`zypper addrepo` exits non-zero when the repository already exists... tried not to write huge checker code for those cases here